cpython/patches/0005-add-emscripten-host.patch

```diff
@@ -10,25 +18,25 @@
      os=-$maybe_os
      basic_machine=`echo "$1" | sed 's/^\(.*\)-\([^-]*-[^-]*\)$/\1/'`
      ;;
-@@ -377,6 +378,7 @@
+@@ -377,6 +378,7 @@ case $basic_machine in
  	| alpha64-* | alpha64ev[4-8]-* | alpha64ev56-* | alpha64ev6[78]-* \
  	| alphapca5[67]-* | alpha64pca5[67]-* | arc-* | arceb-* \
  	| arm-*  | armbe-* | armle-* | armeb-* | armv*-* \
 +	| wasm32 \
  	| avr-* | avr32-* \
  	| ba-* \
  	| be32-* | be64-* \
-@@ -527,6 +529,9 @@
+@@ -527,6 +529,9 @@ case $basic_machine in
  	asmjs)
  		basic_machine=asmjs-unknown
  		;;
 +	wasm32)
 +		basic_machine=wasm32-unknown
```

Comment:
We should really contribute this upstream. Except that I just looked into it and it looks like this file had some major changes in CPython `main`, so I would probably have to try to build Pyodide there just to make sure the patch is correct.